package.json

```diff
@@ -10,15 +10,18 @@
     "name": "Marcel Hebing"
   },
   "dependencies": {
-    "bootstrap": "^3.3.7",
+    "@fortawesome/fontawesome-free": "^5.9.0",
+    "bootstrap": "^4.3.1",
     "d3": "^3.5.17",
     "datatables.net": "^1.10.19",
-    "datatables.net-bs": "^2.1.1",
+    "datatables.net-bs4": "1.10.19",
+    "datatables.net-buttons-bs4": "^1.5.6",
     "datatables.net-dt": "^2.1.1",
     "datatables.net-responsive": "^2.2.3",
     "datatables.net-responsive-bs": "^2.2.3",
```

Comment:
This dependency is no longer used, right?

Reply:
Maybe try to remove as many datatables.net-* dependencies and see if it still works alright?